grouper/fe/settings.py

```diff
@@ -15,16 +15,16 @@ def override_timezone(self, value):
 
 
 settings = FeSettings.from_settings(base_settings, {
+    "address": None,
+    "cdnjs_prefix": "//cdnjs.cloudflare.com",
+    "date_format": "%Y-%m-%d %I:%M %p",
     "debug": False,
+    "how_to_get_help": None,
     "num_processes": 1,
     "port": 8989,
-    "address": None,
-    "timezone": FeSettings.default_timezone,
-    "date_format": "%Y-%m-%d %I:%M %p",
-    "cdnjs_prefix": "//cdnjs.cloudflare.com",
-    "user_auth_header": "X-Grouper-User",
     "refresh_interval": 60,
-    "url": "http://127.0.0.1:8888",
-    "how_to_get_help": None,
     "site_docs": None,
+    "timezone": FeSettings.default_timezone,
+    "url": "http://127.0.0.1:8888",
+    "user_auth_header": "X-Grouper-User",
```

Comment:
OK :) It'd be awesome if this could be broken out into a separate commit so future folks don't get confused :)